web/wp-config.php

```diff
@@ -14,180 +14,75 @@
  */
 require_once( $rootPath . '/vendor/autoload.php' );
 
-/*
- * Fetch .env
- */
-if ( ! isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && file_exists( $rootPath . '/.env' ) ) {
-	$dotenv = Dotenv\Dotenv::create($rootPath);
-	$dotenv->load();
-	$dotenv->required( array(
-		'DB_NAME',
-		'DB_USER',
-		'DB_HOST',
-	) )->notEmpty();
-}
-
-/**
- * Disallow on server file edits
- */
-define( 'DISALLOW_FILE_EDIT', true );
-define( 'DISALLOW_FILE_MODS', true );
-
 /**
  * Force SSL
  */
 define( 'FORCE_SSL_ADMIN', true );
 
 /**
- * Limit post revisions
- */
-define( 'WP_POST_REVISIONS', 3 );
-
-/*
- * If NOT on Pantheon
+ * Pantheon platform settings. Everything you need should already be set.
  */
-if ( ! isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ):
-	/**
-	 * Define site and home URLs
-	 */
-	// HTTP is still the default scheme for now.
-	$scheme = 'http';
-	// If we have detected that the end use is HTTPS, make sure we pass that
-	// through here, so <img> tags and the like don't generate mixed-mode
-	// content warnings.
-	if ( isset( $_SERVER['HTTP_USER_AGENT_HTTPS'] ) && $_SERVER['HTTP_USER_AGENT_HTTPS'] == 'ON' ) {
-		$scheme = 'https';
-	}
-	$site_url = getenv( 'WP_HOME' ) !== false ? getenv( 'WP_HOME' ) : $scheme . '://' . $_SERVER['HTTP_HOST'] . '/';
-	define( 'WP_HOME', $site_url );
-	define( 'WP_SITEURL', $site_url . 'wp/' );
+if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+	require_once(dirname(__FILE__) . '/wp-config-pantheon.php');
```

Comment:
I think our minimum PHP version allows us to just use __DIR__ here.

Reply:
Could this be an exact copy of wp-config.php from panthoen-systems/wordpress? That would be preferable.

Reply:
@greg-1-anderson well, this is an exact copy of panthoen-systems/wordpress as per https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php#L14-L15

Reply:
I meant the whole file. See other comments.